src/DataGrid.tsx

```diff
@@ -568,6 +569,19 @@ function DataGrid<R, SR, K extends Key>(
       }
     }
 
+    if (column.resizable && isCtrlKeyHeldDown(event) && event.shiftKey) {
```

Comment:
need some insights here @amanmahajan7 @nstepien 
1. should we provide 'resizeViaKeyboard' flag? so that user can turn it on and off?
2. what is the best way to provide handler via API? not sure if I can use `onCellKeyDown`

Reply:
I decided to go with using the `onCellKeyDown` API. but still not sure as this doesn't seem to be very intrinsic

Reply:
I think we can always allow keyboard resizing if the column can be resized. I don't think we need an api to disable it unless there is a use case. We allow keyboard navigation on the header cell and can allow resizing when the cell is focused. 

- Check if the cell is focused and column is resizable 
- Check if a particular key combination is pressed (Ctrl + arrow right/left?)

Questions:
- Which key to use. Are there any grid examples we can find?
- Do we need some sort of timer if they keep combination is pressed and increment the width by 1px every few ms?